pkgs/games/julius/default.nix

```
@@ -5,8 +5,15 @@
 , SDL2_mixer
 , cmake
 , libpng
+, runCommand
+, darwin ? null
```

```suggestion
, darwin
```
that will never be null, even on linux

---

Good to know, thanks!